test/assert/tekton_config.go

```diff
@@ -0,0 +1,39 @@
+package assert
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	config "github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type TektonConfigAssertion struct {
```

Comment:
do we really need this assertion?
The `Config` is either created by us as part of the test setup or by the OSP controller, but in that case, we don't have to check the existence (when everything else is running)